doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -718,6 +718,24 @@ the toplevel, and using them in source files is discouraged.
    the :ref:`command line option <command-line-options>` :n:`-I @string` or by the command :cmd:`Add
    ML Path` `@string` (cf. :cmd:`Declare ML Module`).
 
+.. _extra_dependencies:
+
+Extra Dependencies
+------------------
+
+Dependencies on external files, i.e. non ``.v`` files, can be declared as
+follows:
+
+.. cmd:: From @dirpath Extra Dependency @string {? as @ident }
+
+Coq checks that a file name :n:`@string` exists in one, and only one, directory
+associated to :n:`@dirpath`. If :n:`@ident` is provided then Coq gives
+access to the full path of the external file to OCaml plugins via the
+``ComExtraDeps.query_extra_dep`` API.
+
+The ``coqdep`` tool generates a dependency on the external file for the
+current ``.v``.
```

Comment:
If (and only if) you keep the deprecated compatibility syntax (👍 ) it should also be documented.

Reply:
done